onnxruntime/core/providers/qnn/builder/qnn_utils.h

```diff
@@ -26,6 +27,22 @@ class QnnOpConfigWrapper;
 class QnnModelWrapper;
 
 namespace utils {
+/**
+ * Returns a lowercase version of the input string.
+ * /param str The string to lowercase.
+ * /return The lowercased string.
+ */
+inline std::string GetLowercaseString(std::string str) {
```

Comment:
Note: this is a duplicate of the function in core/common/string_utils.h (which is internal to ORT). I don't see a need to add this simple utility function to the provider bridge api.